packages/wdio-utils/src/shim.ts

```diff
@@ -99,6 +146,121 @@ let wrapCommand = function wrapCommand<T>(commandName: string, fn: Function): (.
 
         return commandResult
     }
+
+    function wrapElementFn (promise: Promise<Clients.Browser>, cmd: Function, args: any[], prevInnerArgs?: { prop: string, args: any[] }): any {
+        return new Proxy(
+            Promise.resolve(promise).then((ctx: Clients.Browser) => cmd.call(ctx, ...args)),
```

Comment:
Would this work as well?
```suggestion
            promise.then((ctx: Clients.Browser) => cmd.call(ctx, ...args)),
```

Reply:
nope, because we don't know if `promise` is actually a promise, so to be sure we wrap it this way